static/web.js

```diff
@@ -358,6 +358,33 @@
         );
     }
 
+    /**
+     * Fetches a code snippet from the [`ponylang/pony-tutorial`](https://github.com/ponylang/pony-tutorial) repository on GitHub
+     * and loads it into the code editor
+     * @param {Ace.EditSession} session (see [ace.Ace.Editor::getSession](https://ajaxorg.github.io/ace-api-docs/interfaces/ace.Ace.Editor.html#getSession))
+     * @param {HTMLDivElement} result 
+     * @param {String} gist_id 
+     * @param {bool} do_evaluate 
+     * @param {HTMLButtonElement} evaluateButton 
+     * @return {void}
+     */
+    function fetchSnippet(session, result, snippet_file_name, do_evaluate, evaluateButton) {
+        session.setValue("// Loading snippet: https://github.com/ponylang/pony-tutorial/blob/main/code-samples/" + snippet_file_name + " ...");
+        httpRequest("GET", "https://raw.githubusercontent.com/ponylang/pony-tutorial/" + (query.get("branch") ?? "main") + "/code-samples/" + snippet_file_name, null, 200,
```

Comment:
In the first line here you have hard-coded `main` as the branch in the `session.setValue` string, but in the second line you are using `query.get("branch") ?? "main"`

Can you please make the two match (either both hard-coded as `main`, or both taking their value from the query param)?

Reply:
As I understand it, the branch is meant for the compiler from the ponyc repository, right? So me using it for the snippets from the pony-tutorial repository was a mistake, then and I would revert my use of the variable there if my assumption is indeed right.

Reply:
It looks to me like the `branch` query param isn't truly being used for... anything? It may be leftover from the code we lifted from Rust, without providing any value.

So I'd suggest just hard-coding `main` for both here, I say.

Reply:
Okay, so I did a little digging concerning the evaluation including the `branch` query parameter:

It is used within the API routes:
https://github.com/ponylang/pony-playground/blob/a08532f9fbb87a9e445bafce6c7ab40be95ea6c1/src/routes.rs#L17-L20
https://github.com/ponylang/pony-playground/blob/a08532f9fbb87a9e445bafce6c7ab40be95ea6c1/src/routes.rs#L24-L27
https://github.com/ponylang/pony-playground/blob/a08532f9fbb87a9e445bafce6c7ab40be95ea6c1/src/routes.rs#L48-L52
https://github.com/ponylang/pony-playground/blob/a08532f9fbb87a9e445bafce6c7ab40be95ea6c1/src/routes.rs#L57-L60

However, from there, I can't see it being used anywhere within the playpen crate:
https://github.com/ponylang/pony-playground/blob/main/src/bin/playpen.rs

But I don't know Rust very well. I might have missed something.

In case, I haven't, the potential PR concerning the removal of the `branch` query parameter would not only involve the web.js file but also the routes.rs file. Just so you know.